exe-unit/components/http-proxy/src/gsb_to_http.rs

```diff
@@ -0,0 +1,106 @@
+use crate::headers::Headers;
+use crate::message::GsbHttpCallMessage;
+use crate::response::GsbHttpCallResponseEvent;
+use async_stream::stream;
+use chrono::Utc;
+use futures_core::stream::Stream;
+use tokio::sync::mpsc;
+
+#[derive(Clone, Debug)]
+pub struct GsbToHttpProxy {
+    pub base_url: String,
+}
+
+impl GsbToHttpProxy {
+    pub fn pass(
+        &mut self,
+        message: GsbHttpCallMessage,
+    ) -> impl Stream<Item = GsbHttpCallResponseEvent> {
+        let url = format!("{}{}", self.base_url, message.path);
+
+        let (tx, mut rx) = mpsc::channel(24);
+
+        tokio::task::spawn_local(async move {
+            let client = reqwest::Client::new();
+
+            let method = match message.method.to_uppercase().as_str() {
+                "POST" => actix_http::Method::POST,
+                "GET" => actix_http::Method::GET,
+                _ => actix_http::Method::GET,
+            };
+            let mut builder = client.request(method, &url);
+
+            builder = Headers::add(builder, message.headers);
+
+            log::info!("Calling {}", &url);
+            let response = builder.send().await;
+            let response = match response {
+                Ok(response) => response,
+                Err(err) => {
+                    panic!("Error {}", err);
+                }
+            };
+
+            log::info!("Got response");
+
+            let bytes = response.bytes().await.unwrap();
+
+            let str_bytes = String::from_utf8(bytes.to_vec()).unwrap();
```

Comment:
I'm not sure if it is a good idea. Is image convertible to utf8 string?
And in general the content can contain anything. We should have bytes array instead.